src/evaluate.py

```diff
@@ -0,0 +1,99 @@
+from typing import Dict
```

Comment:
I would propose to put this in a `model` directory (for now inside of the already existing `model` directory, but that will be renamed later)